lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1475,30 +1494,32 @@ private FoundHits queryHits(
 
         Response resp = makeRequest("GET", urlPath, ImmutableMap.copyOf(doc));
         @SuppressWarnings("unchecked")
+//        String results = IOUtils.toString(resp.getEntity().getContent(), "UTF-8");
         Map<String, Object> data = UObject.getMapper().readValue(
                 resp.getEntity().getContent(), Map.class);
         ret.guids = new LinkedHashSet<>();
         @SuppressWarnings("unchecked")
         Map<String, Object> hitMap = (Map<String, Object>) data.get("hits");
         ret.total = (Integer)hitMap.get("total");
         if (loadObjects) {
-            ret.objects = new ArrayList<ObjectData>();
+            ret.objects = new ArrayList<>();
         }
         @SuppressWarnings("unchecked")
         List<Map<String, Object>> hitList = (List<Map<String, Object>>) hitMap.get("hits");
         for (Map<String, Object> hit : hitList) {
             @SuppressWarnings("unchecked")
             Map<String, Object> obj = (Map<String, Object>) hit.get("_source");
+            Map<String, ArrayList> highlightRes = (Map<String, ArrayList>) hit.get("highlight");
```

Comment:
No non-parameterized Generics

Reply:
could you tell me how is this a non-parameterized Generic? B/c of the Map?

Reply:
ArrayList is a generic class, and it's not parameterized.